module-4/backend/week_two.md

```diff
@@ -5,18 +5,54 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+
+ES6 introduces syntax sugar: arrow functions, Classes
+
 2. What's the difference between asynchronous and synchronous JavaScript? 
+
+Synchronous JS can only run one thread of operations, asynchronous JS can run multiple threads thanks to the browser through the event loop
+
 3. What are the four pillars of Object Oriented programming?
+
+Encapsulation, Inheritance, Polymorphism and Abstraction
+
 4. What are some tools available in JavaScript to help you write object oriented code?
+
+ES6: Class syntax
```

Comment:
Remember that `class` objects compile down to the same collection of these ES5 properties.